src/screens/application/container/applicationContainer.js

```diff
@@ -38,6 +38,7 @@ import {
 import { getUser, getPost } from '../../../providers/steem/dsteem';
 import { switchAccount } from '../../../providers/steem/auth';
 import { setPushToken } from '../../../providers/esteem/esteem';
+import NavigationService from '../../../navigation/service';
```

Comment:
why did u use as all func instead using like this `{ navigate }`

Reply:
for easy readability. when using navigate it can be mixed with navigation navigate method

Reply:
but you only used `navigate`